### PR TITLE
update usage message on qbamfilter

### PR DESCRIPTION
**Pull request by ChristinaXu2017**
# Description

Updating qbamfilter documentation (readthedocs.io) and messages.properties file. Hence it will show the same tool usage message when using the command "java -jar qbamfilter.jar -h" and reading .readthedocs.io 
Fixes # (issue)

## Type of change

Documentation
# How Has This Been Tested?
No testing required

# Are WDL Updates Required?
The options keep the same, there is no need to update WDL. 

# Checklist:

n/a

### CODE REVIEW

**Comment by grendeloz** [comment]
There are a few typos - DESCRITPION should be DESCRIPTION in a few places.

Also, there are a lot more messages being deleted than replaced. Are these no longer needed?  Otherwise looks good.

**Comment by ChristinaXu2017** [comment]
typo of "DESCRITPION" is fixed. There were many old messages removed due to dated and unused.

File: docs/qbamfilter/index.md [hunk 1]
```diff
@@ -15,33 +15,38 @@ There is a separate page with a more detailed explanation of the
...
+* **To do a build of qsignature, first clone the adamajava repository using "git clone":**
+  ```
+  git clone https://github.com/AdamaJava/adamajava
+  ```
+  
+  Then move into the adamajava folder:
+  ```
+  cd adamajava
+  ```
+  Run gradle to build qsignature and its dependent jar files:
```
Comment (holmeso) [inline docs/qbamfilter/index.md:34]: `qsignature` should be `qbamfilter`

File: docs/qbamfilter/index.md [hunk 2]
```diff
@@ -15,33 +15,38 @@ There is a separate page with a more detailed explanation of the
...
+qbamfilter requires java 8 and (ideally) a multi-core machine, although 
 it operates in single-threaded mode by default.  You can tune the amount
 of memory used by qbamfilter by specifying the number of records to store 
 in memory (`--maxRecordNumber`).  You can also opt to sort the output BAM 
 and the BAM will be automatically indexed if the sort-by-coordinate option
 is specified.
 
-* Download the [qbamfilter tar file](http://sourceforge.net/projects/adamajava/files/qbamfilter.tar/download)
-* Untar the tar file into a directory of your choice
+* **To do a build of qsignature, first clone the adamajava repository using "git clone":**
```
Comment (holmeso) [inline docs/qbamfilter/index.md:25]: `qsignature` should be `qbamfilter`

Comment (ChristinaXu2017) [reply docs/qbamfilter/index.md]: fixed, thanks

File: docs/qbamfilter/index.md (hunk 1 above)
Comment (ChristinaXu2017) [reply docs/qbamfilter/index.md]: fixed

**Review by holmeso — APPROVED** [review]

**Pull request merged**